shepherd

```diff
@@ -21,13 +21,13 @@ update_services() {
     if [[ " $blacklist " != *" $name "* ]]; then
       image_with_digest="$(docker service inspect "$service" -f '{{.Spec.TaskTemplate.ContainerSpec.Image}}')"
       image=$(echo "$image_with_digest" | cut -d@ -f1)
-      echo "Trying to update service $name with image $image"
+      [ "$NO_VERBOSE" = "true" ] || echo "Trying to update service $name with image $image"
```

Comment:
- [ ] We need to prevent an error if this variable is not set, so it should probably be `[ "${NO_VERBOSE:-}" = "true" ]` instead.